scrapy/pipelines/images.py

```diff
@@ -127,15 +129,37 @@ def get_images(self, response, request, info):
             raise ImageException("Image too small (%dx%d < %dx%d)" %
                                  (width, height, self.min_width, self.min_height))
 
-        image, buf = self.convert_image(orig_image)
+        if self._deprecated_convert_image is None:
+            self._deprecated_convert_image = 'response_body' not in get_func_args(self.convert_image)
+            if self._deprecated_convert_image:
+                from scrapy.exceptions import ScrapyDeprecationWarning
+                import warnings
```

Comment:
Could you please move these import to the top level, as well as imports in convert_image?